src/runtime/virtcontainers/pkg/annotations/annotations.go

```diff
@@ -65,6 +68,8 @@ const (
 	// ImageHash is an sandbox annotation for passing a container guest image SHA-512 hash value.
 	ImageHash = kataAnnotHypervisorPrefix + "image_hash"
 
+	IgvmHash = kataAnnotHypervisorPrefix + "igvm_hash"
```

Comment:
nit" seems to miss a comment. And: Is this igvm_hash actually used?

Reply:
Comment added, and this is used in asset.go